templates/shared/user/profile_big_avatar.tmpl

```diff
@@ -24,7 +24,19 @@
 	<div class="extra content gt-word-break">
 		<ul>
 			{{if .ContextUser.Location}}
-				<li>{{svg "octicon-location"}} {{.ContextUser.Location}}</li>
+				<li>
+					{{svg "octicon-location"}}
+					<span>{{.ContextUser.Location}}</span>
+					{{/* Checking whether .LocationQueryURL is empty is arguably excessive, but makes this functionality foolproof */}}
+					{{if and .EnableLocationButton .LocationQueryURL}}
+						{{/* We presume that the LocationQueryURL is safe, as it is provided by the site administrator. */}}
+						<a href="{{.LocationQueryURL | Safe}}{{.ContextUser.Location}}" rel="nofollow noreferrer">
+							<i class="ui right" data-tooltip-content="{{.locale.Tr "user.redirect_to_osm"}}">
```

Comment:
The `<i>` is redundant, these attributes could be on the `<a>`

Reply:
Not really, the icon turns blue in that case and I felt like using additional parameters to disable that was too much.

![image](https://github.com/go-gitea/gitea/assets/30193966/e7ed53da-c283-4746-9d1c-e07876874bfe)

Reply:
Add CSS class "muted" for `<a>`

Reply:
I intended to avoid that for consistency, but there is a reason why I put the <i> tags later anyways (because the tooltip changes, but I think I can improve this in a later PR).